content/data-loading.md

```diff
@@ -0,0 +1,556 @@
+
+```js
+Meteor.publish('lists/private', function() {
+  if (!this.userId) {
+    return this.ready();
+  }
+
+  return Lists.find({userId: this.userId});
+});
+```
+
+What this means is that the above publication can be confident (thanks to Meteor's accounts system) that it will only ever publish private lists to the user that they belong to. Note also that the publication will re-run if the user logs out (or back in again), which means that the published set of private lists will reflect this.
+
+In the case of a non-logged in user, we explicitly call `this.ready()`, which indicates to the subscription that we've sent all the data we are initially going to send (in this case none).
+
+The other type of publication argument is a simple named argument:
+
+```js
+Meteor.publish('list/todos', function(listId) {
+  check(listId, String);
+
+  ...
+});
+```
+
+When we create a subscription to this piblication on the client, we can provide this argument via the `Meteor.subscribe()` call:
+
+```js
+Meteor.subscribe('list/todos', list._id);
+```
+
+Also note that we are careful to check the `listId` is a string as we expect, as described in the Security article.
+
+### Organizing Publications
+
+It makes sense to place a publication in a package alongside the feature that it's targeted. For instance, sometimes publications provide very specific data that's only really useful for the view that they are developed for. In that case, placing the publication in the same package as the view code makes perfect sense.
+
+Often, however, a publication is more general. For example in the Todos example application, we create a `list/todos` publication, which publishes all the todos in a list. Although in the application we only use this in one place (in the `listsShow` template), in a larger app, there's a good chance we might need to access all the todos for a list in other places. So putting the publication in the `todos` package is a sensible approach.
+
+## Using publications: Subscriptions
+
+To use publications, you need to create a subscription to it on the client. To do so, you call `Meteor.subscribe()` with the name of the publication. When you do this, it opens up a subscription to that publication, and the server starts sending data down the wire to ensure that your client collections contain up to date copies of the data that's pushed by the publication.
+
+### Organizing Subscriptions
+
+It is best to place the subscription as close as possible to the place where the data from the subscription is needed. This reduces "action at a distance" and makes it easier to understand the flow of data through your application.
```

Comment:
It's not clear here why "action at a distance" might happen - I think we should call out that making a subscription means potentially all queries on that collection could get new data.